aip/general/0121.md

```diff
@@ -77,12 +77,13 @@ resource schema for that resource across all methods **must** be the same.
 where "None" indicates that the resource neither **is** nor **is contained** in
 the request or the response*
 
-A resource **must** support at minimum [Get][]: clients must be
+- A resource **must** support at minimum [Get][]: clients must be
```

Comment:
Please indent the lines within the list (as you have done for the new list item)